amplify/backend/api/rms/schema.graphql

```diff
@@ -2,16 +2,20 @@ type Main @model @auth(rules: [{allow: private, operations: [read]}]) {
     id: ID!
     displayName: String!
     description: String
+    owner: String
+    location: String
+    batch:[String!]
     tags: [String!]
     items: [String!]
 }
 
 type Items @model @auth(rules: [{allow: private, operations: [read]}]) {
     id: ID!
     name: String!
-    owner: String!
```

Comment:
Don't delete the owner field and batch field yet. You haven't changed all the other logic to change the mapping over, so this will likely give errors.